packages/themes/package.json

```diff
@@ -29,7 +29,8 @@
     "@hig/styles": "^0.1.1"
   },
   "peerDependencies": {
-    "react": "^15.4.1 || ^16.3.2"
+    "react": "^15.4.1 || ^16.3.2",
+    "prop-types": "^15.6.1"
```

Comment:
`prop-types` [should be in dependencies](https://github.com/facebook/prop-types#how-to-depend-on-this-package)